backend/db/migrate/20220817011608_create_property_assessments.rb

```diff
@@ -2,7 +2,15 @@ class CreatePropertyAssessments < ActiveRecord::Migration[7.0]
   def change
     create_table :property_assessments do |t|
 
-      t.timestamps
+        t.timestamps
+        t.string:BDRM_COND
+        t.string:BTHRM_STYLE1
+        t.string:BTHRM_STYLE2
+        t.string:BTHRM_STYLE3
+        t.string:KITCHEN_TYPE
+        t.string:KITCHEN_STYLE1
+        t.string:KITCHEN_STYLE2
+        t.string:KITCHEN_STYLE3
```

Comment:
```suggestion
      t.timestamps
      t.string :bdrm_cond
      t.string :bthrm_style1
      t.string :bthrm_style2
      t.string :bthrm_style3
      t.string :kitchen_type
      t.string :kitchen_style1
      t.string :kitchen_style2
      t.string :kitchen_style3
```
Works, but some style suggestions before we merge this in for consistency.